l10n_ch_import_camt054/models/custom_import_stmt.py

```diff
@@ -0,0 +1,50 @@
+from odoo import api, models
+
+import base64
+
+
+class AccountStatementImportCustomCamt053(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+
+    @api.model
+    def _complete_stmts_vals(self, stmts_vals, journal, account_number):
+
+        stmts_vals = super(AccountStatementImportCustomCamt053, self).\
+            _complete_stmts_vals(stmts_vals, journal, account_number)
+
+        list_transactions = stmts_vals[0]['transactions']
+
+        for transaction in list_transactions:
+            if transaction.get('sub_fmly_cd') == 'RRTN':
+                # When a return transaction is foud, it search for the
+                # opposite transaction (same ref).
+                for transactionBis in list_transactions:
+                    if 'ref' in transactionBis \
+                        and transactionBis['ref'] == transaction['ref']\
+                        and transactionBis != transaction:
+
+                        if 'account_id' in transaction:
```

Comment:
Instead of doing this test here, you can do it at the begin of the loop and it will speedup the process.